setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 3.2.0
```

Comment:
>current_version [](http://example.com/codeflow?start=0&length=15)

we should publish 3.3.0 as a final version because we are adding the new layered deployment feature.